{{cookiecutter.module_name|slugify}}/MANIFEST.in

```diff
@@ -3,3 +3,6 @@ include inmanta_plugins/{{cookiecutter.inmanta_module_name}}/py.typed
 recursive-include inmanta_plugins/{{cookiecutter.inmanta_module_name}}/model *.cf
 graft inmanta_plugins/{{cookiecutter.inmanta_module_name}}/files
 graft inmanta_plugins/{{cookiecutter.inmanta_module_name}}/templates
+graft docs
+graft examples
+graft tests
```

Comment:
I think we also want the `README` and perhaps the `CHANGELOG`. I notice the ticket doesn't list what we want. Perhaps we should take that to Slack?